src/types/GradeLevel.ts

```diff
@@ -21,7 +21,7 @@ const toStringMappings: Readonly<Record<GradeLevel, string>> = {
   [GradeLevel.HsFreshman]: '9th Grade',
   [GradeLevel.HsSophomore]: '10th Grade',
   [GradeLevel.HsJunior]: '11th Grade',
-  [GradeLevel.HsSenior]: '12th Grade (High School Senior)',
```

Comment:
Initially I removed this because it wasn't fitting with the left padding. Either way, it seemed redundant to keep 'high school' since the parent checbox has that already.